### DISCUSSION TITLE
Add feed of activities pertaining to my own repos

### DISCUSSION THREAD

**Discussion by joehanna** [community/community#31967] [Other Feature Feedback, Questions, & Ideas]
I frequently need to review recent activities, by myself or my contributors, across all of my repos, and there is no simple way to do this.

It would be great if the new Feed page could be enhanced further to either have:
* a "My Repos" tab with the summary of all activities related to my own repos only and allow that to be the default
* a filter on the new "For you" tab that toggles activities for my repos only

Thanks!

**Comment by eliot-akira** [comment]
This would be useful, to have a feed of activities related to my own repos and organizations.

I'd also love a filter option to show my own activities as others would see it.

**Comment by chapoi** [comment]
THIS! Is all I want on github. A simple, chronological comprehensive list of ALL my commits across ALL repos. Such a basic thing that isn't here.
